src/components/App.js

```diff
@@ -0,0 +1,10 @@
+import Display from './Display';
+import '../App.css';
+
+const App = () => (
+  <div className="App">
```

Comment:
- [ ] Instead of wrapping the components inside `div` to avoid too unnecessary divs, you can use instead [`React.Fragment`](https://reactjs.org/docs/fragments.html)

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png)

Reply:
@didymus707, Job well-done but you didn't attend to this change mentioned by the previous reviewer.

For some reason, you forgot to make use of React Fragment here, kindly use it.